GRDB/Record/FetchableRecord+Decodable.swift

```diff
@@ -79,7 +81,17 @@ private struct RowKeyedDecodingContainer<Key: CodingKey>: KeyedDecodingContainer
             } else if dbValue.isNull {
                 return nil
             } else {
-                return try T(from: RowDecoder(row: row, codingPath: codingPath + [key]))
+                do {
+                    let natural = try T(from: RowDecoder(row: row, codingPath: codingPath + [key]))
+                    return natural
```

Comment:
I think `return try T(from...)` is enough. That "natural" word was useful for communication between us, but has no real purpose in the code.

A comment like the following will help making the code more clear:

```swift
do {
    // This decoding will fail for types that need a keyed container,
    // because we're decoding a database value here (string, int, double, data, null).
    // Support for keyed containers is provided below (JSON decoding).
    return try T(from...)
```

Reply:
We did this because you asked us to "do { try natural decoding } catch { try JSON decoding }" if we just return return try T we are not catching it, did we missunderstand something ?

Reply:
Done